lib/helpers/broadcast.js

```diff
@@ -77,12 +120,49 @@ function parseBroadcastFromContentfulEntry(contentfulEntry) {
     data.topic = null;
     data.message.template = contentfulEntry.fields.template || 'askSignup';
   }
+  return Promise.resolve(data);
+}
+
+/**
+ * @param {Object} contentfulEntry
+ * @return {Object}
+ */
+function parseBroadcastMessageFromContentfulEntryAndTemplateName(contentfulEntry, templateName) {
+  const broadcastMessage = contentfulEntry.fields.broadcast;
+  return helpers.contentfulEntry
+    .getMessageTemplateFromContentfulEntryAndTemplateName(broadcastMessage, templateName);
```

Comment:
One more minor naming suggestion. Calling it `broadcastMessageEntry` or `broadcastEntry` could help imply that we are sending an actual entry object, without having previous knowledge of how these C.io schemas work. For a long time I though we were sending a string, until I consoled it out and realized it was a nested entry!